esmvalcore/preprocessor/_io.py

```diff
@@ -134,6 +157,50 @@ def load(
     return cubes
 
 
+def _load_zarr(
+    file: str | Path | Cube | CubeList | xr.Dataset | ncdata.NcData,
+    ignore_warnings: list[dict[str, Any]] | None = None,
+    backend_kwargs: dict[str, Any] | None = None,
+) -> CubeList:
+    if isinstance(file, Path):
+        zarr_xr = xr.open_dataset(
+            file,
+            consolidated=False,
+            engine="zarr",
+        )
```

Comment:
```suggestion
```

This is the part which is supposed to tread files from the local filesystem, right? You also need to consider the case where `file` is a `str` and points to an object in the local filesystem. Currently, this will just run into the `else` part and return an empty list.

I would actually put this in the `else` condition below (see my comments below) and let xarray raise any potential errors.

Reply:
and it's also tested fully now :beer:

Reply:
Sorry, I cannot find the test. Would you point me to it? 😅

Reply:
Ah, that's a nice test, but not exactly what I meant. I think it would be nice to simply replicate `test_load_zarr2_local` with a `str` as `file` arguement, i.e., simply use `cubes = load(str(zarr_path))` in there.

Reply:
ah that I missed, plopped in 84a33f2e61ed7d3dbe481e4c0c9b9ba56fb289bd